packages/atomic/src/components/atomic-modal/atomic-modal.tsx

```diff
@@ -90,55 +105,55 @@ export class AtomicModal implements InitializableComponent {
     }
 
     return (
-      <div
-        part="backdrop"
-        class={`fixed left-0 top-0 right-0 bottom-0 z-10 ${
-          this.isOpen ? 'opened' : ''
-        }`}
-        onClick={(e) => e.target === e.currentTarget && this.close()}
-      >
-        <atomic-focus-trap
-          active={this.isOpen}
-          role="dialog"
-          aria-modal={this.isOpen.toString()}
-          aria-labelledby={this.headerId}
+      <Host class={this.getClasses().join(' ')}>
+        <div
+          part="backdrop"
+          class="fixed left-0 top-0 right-0 bottom-0 z-10"
```

Comment:
Why not go "all in" on a really huge z-index value ? So that it has more chance to always be on top of whatever UI we're implementing.

Reply:
Sure, will do.